modules/nodes-nodes-cgroups-2.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-working.adoc
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="nodes-nodes-cgroups-2_{context}"]
+= Enabling cgroups version 2
+
+You can enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control groups version 2] (cgroups v2) on specific nodes in your cluster by using a machine config. The {product-title} process for enabling cgroups v2 disables all cgroups version 1 controllers and hierarchies.
+
+[NOTE]
+====
+The {product-title} cgroups version 2 feature is in Developer Preview and is not supported by Red Hat at this time.
```

Comment:
Is this still a Developer Preview?  I am told that we don't doc Developer Previews for a couple of reasons which include support concerns.

Reply:
I thought we didn't. But there are 14 files that reference Developer Preview features. We address Developer Preview features [here](https://docs.openshift.com/container-platform/4.9/rest_api/understanding-compatibility-guidelines.html#api-compatibility-common-terminology-dev-preview_compatibility-guidelines).

Reply:
It might be best to discuss with your CS.

Reply:
@kmccarron-rh we did when I acked this with Michael. Please proceed.